pkg/ruleengine/v1/factory.go

```diff
@@ -21,6 +21,7 @@ func NewRuleCreator() *RuleCreatorImpl {
 			R0008ReadEnvironmentVariablesProcFSRuleDescriptor,
 			R0009EbpfProgramLoadRuleDescriptor,
 			R0010UnexpectedSensitiveFileAccessRuleDescriptor,
+			R0011UnexpectedNetworkTrafficRuleDescriptor,
```

Comment:
I would call this rather "Unexpected Egress Network Traffic", we are detecting only egress.(I am not planning internal traffic, even if so, in a different rule)

Reply:
done